src/ol_infrastructure/infrastructure/vault/__main__.py

```diff
@@ -426,6 +449,9 @@ def cloud_init_user_data(
         init_inputs["tls_key"],
         init_inputs["tls_cert"],
         init_inputs["ca_cert"],
+        init_inputs["grafana_api_key"],
+        init_inputs["grafana_prometheus_user"],
+        init_inputs["grafana_loki_user"],
```

Comment:
These will need to be added to the `Output.all` parameters and pulled from the stack configs.